firebase-dataconnect/firebase-dataconnect.gradle.kts

```diff
@@ -0,0 +1,234 @@
+  implementation("com.google.firebase:firebase-components:18.0.0") {
+    exclude(group = "com.google.firebase", module = "firebase-common")
+  }
+  implementation(project(":protolite-well-known-types"))
+  implementation("com.google.firebase:firebase-auth-interop:20.0.0") {
+    exclude(group = "com.google.firebase", module = "firebase-common")
+    exclude(group = "com.google.firebase", module = "firebase-components")
+  }
+  implementation("com.google.firebase:firebase-appcheck-interop:17.1.0") {
+    exclude(group = "com.google.firebase", module = "firebase-common")
+    exclude(group = "com.google.firebase", module = "firebase-components")
+  }
+
+  compileOnly(libs.javax.annotation.jsr250)
+  implementation(libs.grpc.android)
+  implementation(libs.grpc.okhttp)
+  implementation(libs.grpc.protobuf.lite)
+  implementation(libs.grpc.kotlin.stub)
+  implementation(libs.grpc.stub)
+  implementation(libs.protobuf.java.lite)
+  implementation(libs.protobuf.kotlin.lite)
+
+  testCompileOnly(libs.protobuf.java)
+  testImplementation(project(":firebase-dataconnect:testutil"))
+  testImplementation(libs.androidx.test.junit)
+  testImplementation(libs.kotest.assertions)
+  testImplementation(libs.kotest.property)
+  testImplementation(libs.mockk)
+  testImplementation(libs.robolectric)
+  testImplementation(libs.truth)
+  testImplementation(libs.truth.liteproto.extension)
+  testImplementation(libs.kotlin.coroutines.test)
+  testImplementation(libs.kotlinx.serialization.json)
+
+  androidTestImplementation(project(":firebase-dataconnect:androidTestutil"))
+  androidTestImplementation(project(":firebase-dataconnect:connectors"))
+  androidTestImplementation(project(":firebase-dataconnect:testutil"))
+  androidTestImplementation("com.google.firebase:firebase-auth:22.3.1") {
+    exclude(group = "com.google.firebase", module = "firebase-common")
+    exclude(group = "com.google.firebase", module = "firebase-components")
+  }
+  androidTestImplementation("com.google.firebase:firebase-appcheck:18.0.0") {
+    exclude(group = "com.google.firebase", module = "firebase-common")
+    exclude(group = "com.google.firebase", module = "firebase-components")
+  }
+  androidTestImplementation(libs.androidx.test.core)
+  androidTestImplementation(libs.androidx.test.junit)
+  androidTestImplementation(libs.androidx.test.rules)
+  androidTestImplementation(libs.androidx.test.runner)
+  androidTestImplementation(libs.kotlin.coroutines.test)
+  androidTestImplementation(libs.kotest.assertions)
+  androidTestImplementation(libs.kotest.property)
+  androidTestImplementation(libs.mockk)
+  androidTestImplementation(libs.mockk.android)
+  androidTestImplementation(libs.truth)
+  androidTestImplementation(libs.truth.liteproto.extension)
+  androidTestImplementation(libs.turbine)
+}
+
+tasks.withType<KotlinCompile>().all {
+  kotlinOptions {
+    freeCompilerArgs = listOf("-opt-in=kotlin.RequiresOptIn")
+  }
+}
+
+extra["packageName"] = "com.google.firebase.dataconnect"
```

Comment:
Do you need this?

Reply:
Good point. I've now removed the `extra["packageName"]` definition; I see it's not longer needed as of https://github.com/firebase/firebase-android-sdk/pull/5895.